### DISCUSSION TITLE
Cannot use deployed Prompt Flow

### DISCUSSION THREAD

**Discussion by friknsy** [microsoft/sample-app-aoai-chatGPT#806] [Q&A] [answered]
Does anyone have any experience applying deployed prompt flow endpoints? I set all the relevant environment variables, but there is no difference other than the answer still using the model set as the azure open ai endpoint. I wonder if any other settings are needed after setting the environment variables. I would like to express my gratitude in advance.

![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/75009635/c70dd233-d727-49e4-973d-573c7930223b)

**Comment by Tanishk1** [comment]
I have the same question, did you figure it out?

**Reply by friknsy** [reply, marked as answer]
The variable “azure_openai_stream” must be set to false, not true. that worked for my case 😂
